src/server/Room.ts

```diff
@@ -50,61 +54,67 @@ export class Room {
         if (!this.hasClient(toClientId)) {
             return returnError(cb, 'Invalid toClientId - clientId ' + toClientId + ' does not belong in the room');
         } else {
-            if (this.clientSessions[toClientId]) {
-                return this.clientSessions[toClientId].sendMessage(packet, cb);
+            if (this._clientSessions[toClientId]) {
+                return this._clientSessions[toClientId].sendMessage(packet, cb);
             }
         }
     }
 
     // (async)
     // Send Messages from a client
-    public clientSendMessage(fromClientId: string,
-                             toClientId: string,
-                             packet: PacketType,
-                             cb?: CallbackType) {
+    public clientSendMessage(
+        fromClientId: string,
+        toClientId: string,
+        packet: PacketType,
+        cb?: CallbackType) {
 
         if (!this.hasClient(fromClientId) || !this.hasClient(toClientId)) {
             return returnError(cb, 'Invalid fromClientId / toClientId');
         } else {
             this.sendMessage(toClientId,
-                             SessionMessageType.SendMessage,
-                             packet,
-                             cb);
+                SessionMessageType.SendMessage,
+                packet,
+                cb);
         }
     }
 
     public hasClient(clientId: string) {
-
-        return this.clients.indexOf(clientId) >= 0;
+        return this._clients.indexOf(clientId) >= 0;
     }
 
-    // Add Client to Room
-    // @arg clientId Unique Id of client
-    // @arg socket The socket.io object of the new client
-    public addClient(clientId: string,
-                     session: Session) {
+    /**
+     * Add Client to Room
+     * @param clientId Unique Id of client
+     * @param socket The socket.io object of the new client
+     */
+    public addClient(
+        clientId: string,
+        session: Session
+    ) {
```

Comment:
need to standardize whether the closing parenthesis and opening brace ') {' start on the new line or not